connectors/RPA/README.md

```diff
@@ -0,0 +1,5 @@
+# Camunda RPA Connector
+
+The **RPA Connector** allows you to execute Scripts in the Camunda RPA Worker.
+
+Learn more about the RPA Worker in the [Camunda Docs](https://docs.camunda.io/docs/next/components/RPA/getting-started/).
```

Comment:
(optional) Please do not use `/next` links. Please link to the first version where this connector was added to